ross/disk_element.py

```diff
@@ -89,11 +89,21 @@ def __eq__(self, other):
         True
         """
         false_number = 0
-        for i in self.__dict__:
-            try:
-                self.__dict__[i] == other.__dict__[i]
-            except:
-                false_number += 1
+        if other.__class__.__name__ == "DiskElement":
```

Comment:
Hi @fernandarossi, your lastest commit tests just failed, and I think the reason beghind is this line.

Since you are only comparing with the `DiskElement`, when building the `DiskElement6DoF` class it will go directly to the `else` statement, hence returning `False`. A possible solution would be:

```python
if "DiskElement" in other.__class__.__name__:
```